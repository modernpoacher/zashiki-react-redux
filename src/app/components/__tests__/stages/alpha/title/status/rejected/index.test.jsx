import React from 'react'
import snapshotOf, {
  getComponentElement
} from 'react-component-snapshot'

import '@testing-library/jest-dom'

import {
  render
} from '@testing-library/react'

import Rejected from '#zashiki-react-redux/app/components/stages/alpha/title/status/rejected'

describe('#zashiki-react-redux/app/components/stages/alpha/status/title/rejected', () => {
  describe('Always', () => {
    it('renders', () => {
      expect(snapshotOf(getComponentElement(render(
        <Rejected
          title='MOCK TITLE'
        />
      ))))
        .toMatchSnapshot()
    })
  })
})
