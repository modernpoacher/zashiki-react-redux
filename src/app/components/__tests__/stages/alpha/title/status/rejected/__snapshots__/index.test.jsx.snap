// Jest Snapshot v1, https://jestjs.io/docs/snapshot-testing

exports[`#zashiki-react-redux/app/components/stages/alpha/status/title/rejected Always renders 1`] = `
<header
  className="alpha rejected"
>
  <h1>
    MOCK TITLE
  </h1>
</header>
`;
